lib/builders.js

```diff
@@ -0,0 +1,14 @@
+const Source = require('./Source.js')
+const View = require('./View.js')
+
+class ViewBuilder {
+  static fromDataset ({ term, dataset, graph, source }) {
+    const view = new View({ term, dataset, graph, source })
+    view.updateEntities(new Source({ endpointUrl: '' })) // @TODO I send this to access the Source constructor. It's like this in the meantime because of circular dependency
```

Comment:
I still don't know how to avoid circular dependency without breaking backward compatibility. So I send this object to access the constructor.